RocketChat/Rocket.Chat
# [FIX] Read receipts showing first messages of the room as read even if not read by everyone

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

- Turn on read receipts feature
- Create a room and invite people
- Send a message
- All messages will be marked as read until the other invited user reads the room for the first time, afterwards the feature will work as expected

## Further comments
